resources/mediawiki.special/mediawiki.special.preferences.js

```diff
@@ -19,6 +19,18 @@ var $legends = $fieldsets.children( 'legend' )
 	.addClass( 'mainLegend' );
 
 	/**
+	 * Wikia Function - @see UC-206
```

Comment:
Right, because the whole file is wrapped in a function but not indented. I will indent these back!